yoyodyne/models/expert.py

```diff
@@ -32,19 +37,24 @@ class ActionVocabulary:
     start_vocab_idx: int
     target_characters: Set[Any]
```

Comment:
Since it's dependent on maxwell, there's still some uses where youc an pass any form of sequences. Though i realize the variable name is already creating an assumption.

Reply:
Ah, realized why i did that. I don't want this module to be too dependent on maxwell classes, so just using any to make it more generalizable.

Reply:
Technically the edit actions can accept any hashable symbol. So strings, ints, even tuples. So any is a better representation of its coverage given that maxwell is also symbol agnostic.

Reply:
Ah, in that case target_characters is a strange name :) - but lets keep it for now.